plans/rpmlint/README.md

```diff
@@ -39,11 +42,26 @@ No options available
 
 - Rpmlint the upstream packit project
   ```yaml
-  plans:
-    import:
-      url: https://github.com/packit/tmt-plans
-      ref: main
-      name: /plans/rpmlint
+  discover:
+    how: fmf
+    filter: "tag: rpmlint"
+    url: https://github.com/packit/tmt-plans
+    ref: main
+  execute:
+    how: tmt
+  ```
+- Use `rpmlintrc` file
+  ```yaml
+  prepare:
+    - how: shell
+      script: cp ./*.rpmlintrc $TMT_PLAN_DATA/
```

Comment:
As a next step, we might want to allow users to define a custom directory via env. var. any copy the `.rpmlintrc` files ourselves. But this definitely helps.

Reply:
I have tried to design it like that, but it would require something like [this](https://github.com/teemtee/tmt/issues/2572). The issue is that once we are in the `tmt-plans`'s plan/test, we only have access to the current fmf-tree files (not even the full repository)

Reply:
Aha, ok. Thanks for the context.